### PR TITLE
[ML] Code dump for azure-ai-ml 0.1.0b7

### PR DESCRIPTION
**Pull request by harneetvirk**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/ml/azure-ai-ml/dev_requirements.txt [hunk 1]
```diff
@@ -13,4 +16,7 @@ pytest
 pydash
 pytest-vcr
 azure-mgmt-storage~=19.1.0
-pywin32==227 ; sys_platform == 'win32'
+azure-devtools==1.2.0
```
Comment (scbedd) [inline sdk/ml/azure-ai-ml/dev_requirements.txt:19]: Install local version of azure-devtools.

Comment (scbedd) [inline sdk/ml/azure-ai-ml/dev_requirements.txt:19]:
```suggestion
```

Comment (scbedd) [reply sdk/ml/azure-ai-ml/dev_requirements.txt:19]: Left comment with suggestion of fix. you already have the correct relative install present on line 3.

Comment (harneetvirk) [reply sdk/ml/azure-ai-ml/dev_requirements.txt]: The pinned version of azure-devtools has been removed from requirements file.

**Pull request closed without merging**